### ISSUE TITLE
Fspy app window isn't opening

### ISSUE THREAD

**Issue by catisioo** [stuffmatic/fSpy#71]
I used fspy few times without any problems and everything worked great, but for some reason it just stopped working.
(I have a 64 bit Windows 10 laptop)

The problem is that it shows up on my taskbar as an active program and when I hover my mouse over the icon on taskbar, it shows the window like it would show it for any other program. Here's an image, of how it looks:

<img width="231" alt="fspy" src="https://user-images.githubusercontent.com/74970648/100103159-33c82600-2e6d-11eb-9b30-2539056e6c8e.png">

But the fspy window just isn't anywhere. This is very similar to what would be happening, if I had another screen connected and the window would be placed there, but I don't have 2 screens, and my project setting is set to PC screen only. Here's a gif of me switching my programs, you can see when i stop on the fspy tab, nothing appears, as if it was transparent:

![fspygif](https://user-images.githubusercontent.com/74970648/100104344-b9000a80-2e6e-11eb-8270-ec81749796ed.gif)

I have tried restarting my computer, connecting to a second screen, reinstalling the program and I even installed the 1.0.2 version, but the problem is still there.

Judging from this, it's most likely a problem on my PC, but I have no idea how to fix it and I haven't found any alternatives to fspy. I really need fspy for some blender projects I have planned, so any help on how to fix this problem is appreciated

**Comment by Beryesa** [comment]
Can you check screen settings tab if you have two monitors?
I remember that on a laptop that has 2 graphcs card, windows was showing it as second monitor and some apps goes there which that i cant reach.

**Comment by Beryesa** [comment]
Can you try Windows key + shift + left/right after selecting fspy from alt+tab.
![image](https://user-images.githubusercontent.com/60751519/107343230-316f8280-6ad2-11eb-83c2-cc0b31842075.png)

**Comment by catisioo** [comment]
Yeah, I already tried that, It doesn't show up on neither of monitors. (I previously had 1 monitor, now that I have 2, the problem is basically the same)

I'm sure that this is just some stupid bug on my laptop and I'll just try installing a linux VM for using fspy, hopefully it works there

**Comment by FlynnD273** [comment]
Same issue here. Running Windows 10. Stopped working after an update, I think. I've tried all the different compatibility modes, with no success

**Comment by catisioo** [comment]
Update:

For context, I use 2 monitors most of the time. Fspy still has the same problem.

Recently, I unplugged my second monitor and I now use only my laptop. I tried opening VLC media player and now it has the same problem as fspy did. It only opens on a "mystical" second monitor that I apparently have (which I don't. I only have my laptop.) Changing from PC screen only to duplicate or extend doesn't change anything. Pressing windows+shift+left/right also doesn't do anything. I currently don't have access to a second monitor so I can't try anything with that.

I tried to open fspy, as an example of another program that has this problem, but it actually opened on my main screen this time. (it wasn't entirely on my screen, only about 1/3 of the window, but I could easily press the maximize button). What.

Soo, my fspy magically works now. VLC media player does not. I'm guessing that this is some stupid windows bug, since it affects not only fspy, but also another program.

**Comment by jaymulholland** [comment]
hello, coming in to say i experience this issue as well where fSpy is open but not visible. fSpy application is on taskbar but will not open. I use two monitors, and newly updated windows 10. 
i have tried installing / reinstalling. However, i went to try this^, windows key + shift + left / right. I actually pressed up and down as well, and somehow the window appeared and i am able to use fSpy again. 
its clear this is some finicky bug where applications are either not overlapping properly, or cant be accessed. but if it happens to anyone else, try this.

**Comment by jessaltorres** [comment]
Hi, I'm having the same problem but im working on an iMac. It's updated to Sonoma 14.4.1. My business now depends on fSpy. Can anyone help me out please?

**Comment by catisioo** [comment]
> Hi, I'm having the same problem but im working on an iMac. It's updated to Sonoma 14.4.1. My business now depends on fSpy. Can anyone help me out please? 

I worked around it by running a Linux Ubuntu virtual machine and sending my picture there, doing the perspective matching and then sending the file back to my main computer through an email.
I don't know if Mac devices have free vm software available like oracle vm virtual box (which I used). You could also try to dual boot Linux from a USB or something.

Super convoluted solution, but it works as a last resort.

**Comment by ElioMacho** [comment]
I have the same issue (Mac user) any solution around?

**Comment by myko-ll** [comment]
same issue myko ll I've did also possible but nothing but I had backed fspy on one of my drives I think using an older versions can help b4 it automatically updates let me hope it does'nt
